package.json

```diff
@@ -50,6 +50,7 @@
         "lodash-es": "4.17.21",
         "mobile-drag-drop": "2.3.0-rc.2",
         "ngx-device-detector": "3.0.0",
+        "ngx-infinite-scroll": "13.0.0",
```

Comment:
Just to make sure: This was discussed with Stephan, right?
I see that it is needed for this feature, but new dependencies should be discussed with Stephan first.